content/en/docs/tasks/administer-cluster/sysctl-cluster.md

```diff
@@ -100,25 +102,36 @@ The following sysctls are known to be _namespaced_:
 - `fs.mqueue.*`,
 - `net.*`.
 
-Sysctls which are not namespaced are called _node-level_ and must be set
-manually by the cluster admin, either by means of the underlying Linux
-distribution of the nodes (e.g. via `/etc/sysctls.conf`) or using a DaemonSet
-with privileged containers.
+Sysctls with no namespace are called _node-level_ sysctls. If you need to set
+them, you must manually configure them on the operating system of the underlying
+node or by using a DaemonSet with privileged containers.
 
-The sysctl feature is an alpha API. Therefore, sysctls are set using annotations
-on pods. They apply to all containers in the same pod.
+In general, use the pod security context on pods to configure sysctls.
```

Comment:
Maybe "When possible, use the pod security context to configure sysctls on pods. This context applies to all containers on a given pod."

Reply:
"when possible" is misleading. The pod security context is the way to configure sysctls. There is no alternative.